packages/buefy-next/src/components/progress/ProgressBar.vue

```diff
@@ -43,21 +47,28 @@ export default {
         }
     },
     computed: {
+        parentMax() {
+            return (this.parent as ProgressInstance).max
+        },
```

Comment:
I would rather do
```ts
parentProgress() {
    return this.parent as ProgressInstance
}
```
and replace all the occurrences of `this.parent` with `this.parentProgress`.

Reply:
I meant `(this.parent as ProgressInstance)` → `this.parentProgress`.